_data/tagging-status.yml

```diff
@@ -268,3 +269,13 @@
    status: supported
    updated: 2024-07-05
 
+ - name: KOMA-classes
+   type: class
+   status: currently-unsupported
+   related-issues: [88]
+
+ - name: memoir
+   type: class
+   status: currently-unsupported
+   comments: "Some code showing the needed adjustments can be found in the 
+              [bible example](https://github.com/latex3/tagging-project/blob/main/project-examples/ASV/bible.tex)"
```

Comment:
is that something the memoir maintainer has to do? Should we say that?

Reply:
Yes, but I think we must first review the sec and toc code so that we can tell him which sockets he should insert ...

Reply:
ok